packages/ui/src/components/Visualization/Canvas/Canvas.tsx

```diff
@@ -186,7 +186,7 @@ export const Canvas: FunctionComponent<PropsWithChildren<CanvasProps>> = (props)
   useEffect(() => {
     const timeoutId = setTimeout(() => {
       controller.getGraph().fit(80);
-    }, 250);
+    }, 500);
```

Comment:
Increased the graph alignment timeout, because the resizable sidebar takes a bit more time to be displayed and 250 miliseconds was not enough in most cases